CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ### Added
 
 ### Changed
+- Bumped the version of Node used by Travis to v6
+- Removing custom grid variables.
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2707/commits/26d256eca88e4b45e7b5eda733743e4333b4464f.